### PR TITLE
CORE-19533 - Conditionally set search path only if running in postgresql.

### PR DESCRIPTION
**Pull request by driessamyn**

### CODE REVIEW

File: testing/db-testkit/src/main/kotlin/net/corda/db/testkit/DatabaseInstaller.kt [hunk 1]
```diff
@@ -125,10 +127,26 @@ class DatabaseInstaller(
                 connection.createStatement().use { stmt ->
                     val sql = """
                         CREATE SCHEMA IF NOT EXISTS $schemaName;
-                        SET search_path TO $schemaName;
                     """.trimIndent()
                     stmt.execute(sql)
                 }
+
+                val isPostgres = connection.createStatement().use { stmt ->
```
Comment (kyriathar) [inline testing/db-testkit/src/main/kotlin/net/corda/db/testkit/DatabaseInstaller.kt]: Could we use `net.corda.db.testkit.DbUtils.isInMemory` instead to determine the DB type?

Comment (driessamyn) [reply testing/db-testkit/src/main/kotlin/net/corda/db/testkit/DatabaseInstaller.kt]: I didn't know we had that.
It's not the best way of checking as you can directly use a particular DB helper, irrespective of the gradle property. 
However, I've changed it anyway.

**Review by dickon — APPROVED** [review]
LGTM

**Review by kyriathar — APPROVED** [review]
LGTM

**Pull request merged**